src/components/transport_manager/include/transport_manager/transport_adapter/transport_adapter_impl.h

```diff
@@ -82,9 +82,16 @@ class TransportAdapterImpl : public TransportAdapter,
   TransportAdapterImpl(DeviceScanner* device_scanner,
                        ServerConnectionFactory* server_connection_factory,
                        ClientConnectionListener* client_connection_listener,
-                       resumption::LastState& last_state,
+                       resumption::LastStateWrapperPtr last_state_wrapper,
                        const TransportManagerSettings& settings);
 
+  DEPRECATED
+  TransportAdapterImpl(DeviceScanner* device_scanner,
+                       ServerConnectionFactory* server_connection_factory,
+                       ClientConnectionListener* client_connection_listener,
+                       resumption::LastState& last_state_wrapper,
+                       const TransportManagerSettings& settings) = delete;
```

Comment:
Include proper function definition

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
Still no definition, see https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r372663000 for an example

Reply:
@jacobkeeler Please take a look https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r373481596